Landing-page.css

```diff
@@ -624,81 +589,82 @@ SVG Icons - svgicons.sparkk.fr
 * TEXT - size
 **/
 
-
 .text-sm {
-    font-size: 0.9rem;
+  font-size: 0.9rem;
 }
 
 .text-md {
-    font-size: 1.2rem;
+  font-size: 1.2rem;
 }
 
 .text-lg {
-    font-size: 1.5rem
+  font-size: 1.5rem;
 }
 
 .text-xl {
-    font-size: 2.0rem;
+  font-size: 2rem;
 }
 
 /** 
 * TEXT - color
 **/
 
 .text-white {
-    color: #fff;
+  color: #fff;
 }
 
 .text-black {
-    color: #000;
+  color: #000;
 }
 
 .text-center {
-    text-align: center;
+  text-align: center;
 }
 
+.footer-text-bold {
+  font-style: normal;
+  font-weight: bold;
+  font-size: 36px;
+  line-height: 49px;
+}
 /**
 ** MEDIA QUERIES
 **/
-@media(max-width: 1075px) {
-    header .header-rt a.btn-md {
-        padding: .5rem !important;
-        text-align: center;
-    }
+@media (max-width: 1075px) {
+  header .header-rt a.btn-md {
+    padding: 0.5rem !important;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Use of !important](https://app.codacy.com/gh/codeclannigeria/CodeClanUI/pullRequest?prid=5575170)